flow/layers/transform_layer.cc

```diff
@@ -11,6 +11,17 @@ TransformLayer::TransformLayer() = default;
 TransformLayer::~TransformLayer() = default;
 
 void TransformLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
+  // Checks (in some degree) that SkMatrix transform_ is valid and initialized.
+  //
+  // If transform_ is uninitialized, this assert may look flaky as it doesn't
+  // fail all the time, and some rerun may make it pass. But don't ignore it and
+  // just rerun the test if this is triggered, since even a flaky failure here
+  // may signify a potentially big problem in the code.
+  //
+  // We have to write this flaky test because there is no reliable way to test
+  // whether a variable is initialized or not in C++.
+  FML_DCHECK(transform_.isFinite());
```

Comment:
The unopt variant is not used in any serious manner when exercising flow. Having uninitialized stuff in the matrix is quite serious. Maybe just terminate the process here.

Reply:
I've changed FML_DCHECK to FML_CHECK.